voxel_grid/src/voxel_grid.cpp

```diff
@@ -202,7 +202,7 @@ namespace voxel_grid {
       printf("Layer z = %u:\n",z);
       for(unsigned int y = 0; y < size_y_; y++){
         for(unsigned int x = 0 ; x < size_x_; x++){
-          printf((getVoxel(x, y, z)) == voxel_grid::MARKED? "#" : " ");
+          printf("%s", (getVoxel(x, y, z) == voxel_grid::MARKED)? "#" : " ");
```

Comment:
You should use "%c".

Reply:
Certainly it is a char type.

Reply:
warning: format ‘%c’ expects argument of type ‘int’, but argument 2 has type ‘const char*’ [-Wformat=]

"#" : " " => '#' : ' '
change

Reply:
OK.